src/Microsoft.AspNetCore.Http.Connections.Client/Internal/LongPollingTransport.cs

```diff
@@ -185,6 +185,24 @@ public Task StartAsync(Uri url, TransferFormat transferFormat)
                 _application.Output.Complete(_error);
 
                 Log.ReceiveStopped(_logger);
+
+                // Send the DELETE request to clean-up the connection on the server, but don't wait for it
+                _ = SendDeleteRequest(pollUrl);
+            }
+        }
+
+        private async Task SendDeleteRequest(Uri pollUrl)
+        {
+            try
+            {
+                Log.SendingDeleteRequest(_logger, pollUrl);
+                var resp = await _httpClient.DeleteAsync(pollUrl);
```

Comment:
response. Abbreviations trigger me.

Reply:
I'm on board with you. For some reason I keep doing it with `resp` though...